sdk/devcenter/developer-devcenter-rest/samples/v1-beta/javascript/sampleCreateEnvironment.js

```diff
@@ -7,9 +7,8 @@ const createClient = require("@azure-rest/developer-devcenter").default;
  */
 async function createEnvironment() {
   // Build client and fetch required parameters
-  const tenantId = process.env.AZURE_TENANT_ID || "<tenant id>";
-  const devCenter = process.env.AZURE_DEVCENTER_NAME || "<devcenter name>";
-  const client = createClient(tenantId, devCenter, new DefaultAzureCredential());
+  const endpoint = process.env.AZURE_ENDPOINT || "<endpoint>";
```

Comment:
Looks like the js samples are not updated.  could you please re-publish samples?

Reply:
Samples should be republished now!